Training Plan/BeginningTraining.md

```diff
@@ -124,9 +144,23 @@
     - Add interface definition
     - Fill in behavior in custom robot map
     - Get value from robot map in Drive Subsystem
+ ### Projects
+  - Make mock subsystem
+## Eighth Week:
+ - Interest splits: Auto, drivetrain, arm, LEDs
+ ### Projects
+  - Basic subsystem with mock motors and sensors
+## Ninth Week:
+ - Interest splits: Auto, drivetrain, arm, LEDs
+ ### Projects
+  - Basic subsystem with mock motors and sensors
+## Tenth Week:
+ - Interest splits: Auto, drivetrain, arm, LEDs
+ ### Projects
+  - Basic subsystem with mock motors and sensors
```

Comment:
More detail here? Matt and I can help fill in details if you can add more direction.